CUR_STREAM_VERS not defined in wavpack 5.0.0

`CUR_STREAM_VERS` is no longer defined in wavpack 5.0.0, leading to a compilation warning:
```
ip/wavpack.c: In function 'wavpack_open':
ip/wavpack.c:214:5: warning: "CUR_STREAM_VERS" is not defined [-Wundef]
 #if CUR_STREAM_VERS > 0x404
     ^~~~~~~~~~~~~~~
```

This could be solved using
```
#if defined CUR_STREAM_VERS && CUR_STREAM_VERS > 0x404
```
but [according to the developers](https://hydrogenaud.io/index.php/topic,111822.msg922950.html#msg922950) it was not intended to be used in the first place.

--- a/configure
+++ b/configure
@@ -256,7 +256,7 @@ check_opus()
 
 check_wavpack()
 {
-	pkg_config WAVPACK "wavpack" "" "-lwavpack"
+	pkg_config WAVPACK "wavpack >= 4.40" "" "-lwavpack"
 	return $?
 }
 

--- a/ip/wavpack.c
+++ b/ip/wavpack.c
@@ -211,9 +211,7 @@ static int wavpack_open(struct input_plugin_data *ip_data)
 		| sf_channels(WavpackGetReducedChannels(priv->wpc))
 		| sf_bits(WavpackGetBitsPerSample(priv->wpc))
 		| sf_signed(1);
-#if CUR_STREAM_VERS > 0x404
 	channel_mask = WavpackGetChannelMask(priv->wpc);
-#endif
 	channel_map_init_waveex(sf_get_channels(ip_data->sf), channel_mask, ip_data->channel_map);
 	return 0;
 }

